util/dashboard/gen_dashboard_entry.py

```diff
@@ -63,17 +62,17 @@ def get_doc_url(base, url):
     """
     assert isinstance(url, str) and len(url) > 0
     if url[0] == '/':
-        return url
+        return '/book' + url
     else:
-        return '/' + base + '/' + url
+        return '/book/' + base + '/' + url
 
```

Comment:
It would be really nice to have unit tests for gen_dashboard_entry.py. Then we could have regression tests for the bug(s) this PR addresses. However, in the interest of time, I think it makes sense to merge this first and do unit tests later.

Reply:
Yeah I agree; I'd like these links to be checked.

My plan is to stop using preprocessors for these things (dashboards, reggen, etc.), instead converting them into markdown and inserting them into doc files with DOCGEN. Then they will be subject to the same link checks as everything else and one'll be able to view then in github etc. This was one of the motivations behind https://github.com/lowRISC/opentitan/pull/17793 .